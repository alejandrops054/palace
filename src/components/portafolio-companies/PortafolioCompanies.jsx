/* Sistema Intelligence 2.0
Módulo Contratos
Versión 1.0
Author: IntregraIT Contratos
Comentarios: La tabla esta incompleta revisar 18/01/2021
*/
import React, { useState } from "react";
import "antd/dist/antd.css";
import { Table } from "antd";
import {
  Col,
  Row,
  Card,
  CardBody,
  CardTitle,
  Button,
  ButtonGroup
} from "reactstrap";

export default function Portafoliocompanies() {
  const columns = [
    {
      title: "Code",
      dataIndex: "code",
      key: "code",
      width: 100,
    },

    {
      title: "Name",
      dataIndex: "name",
      key: "name",
      width: 150,
    },
    {
      title: "Number Identification Fiscal",
      dataIndex: "number",
      key: "number",
      width: 100,
    }
  ];
  const data = [
    {
      code: "CIHC",
      name: "CONTROLADORA IHC S.A DE C.V",
      number: "CIHC121455",
    },
    {
        code: "CIHC",
        name: "CONTROLADORA IHC S.A DE C.V",
        number: "CIHC121455",
      },
      {
        code: "CIHC",
        name: "CONTROLADORA IHC S.A DE C.V",
        number: "CIHC121455",
      },
      {
        code: "CIHC",
        name: "CONTROLADORA IHC S.A DE C.V",
        number: "CIHC121455",
      },
      {
        code: "CIHC",
        name: "CONTROLADORA IHC S.A DE C.V",
        number: "CIHC121455",
      },
      {
        code: "CIHC",
        name: "CONTROLADORA IHC S.A DE C.V",
        number: "CIHC121455",
      },
      {
        code: "CIHC",
        name: "CONTROLADORA IHC S.A DE C.V",
        number: "CIHC121455",
      },
      {
        code: "CIHC",
        name: "CONTROLADORA IHC S.A DE C.V",
        number: "CIHC121455",
      },
  ];
  return (
    <>
      <Card>
        <CardBody>
          <CardTitle>
            <h3>Portafolio companies</h3>
          </CardTitle>          
          <br />
          <span>Catalog of portafolio companies</span>
          <Row>
            <Col xs="12">
              <Table columns={columns} dataSource={data} />
            </Col>
          </Row>
          <Row>
            <Col xs="7">
            </Col>
            <Col xs="5">
            <ButtonGroup>
            <Button color="primary">Log</Button>
            <Button color="primary">Add</Button>
            <Button color="primary">Properties</Button>
            <Button color="primary">Add Like</Button>
            <Button color="primary">Refresh</Button>
            <Button color="primary">Close</Button>
            </ButtonGroup>
            </Col>
          </Row>
        </CardBody>
      </Card>
    </>
  );
}
