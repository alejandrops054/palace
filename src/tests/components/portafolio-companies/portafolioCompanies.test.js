/* Sistema Intelligence 2.0
Módulo Contratos
Versión 1.0
Author: IntregraIT Contratos
Comentarios: xxxxx
*/
import React from 'react';
import '@testing-library/jest-dom'
import { shallow } from 'enzyme'
import PortafolioCompanies from '../../../components/portafolio-companies/PortafolioCompanies'

describe('PortafolioCompanies component',() => {
    test('Debe mostrar snapshot', () => {
        
        const wrapper = shallow(<PortafolioCompanies/>)
        expect(wrapper).toMatchSnapshot();
    })
    
})