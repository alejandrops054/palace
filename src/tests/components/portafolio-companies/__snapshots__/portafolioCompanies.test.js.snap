// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PortafolioCompanies component Debe mostrar snapshot 1`] = `
<Fragment>
  <Card
    tag="div"
  >
    <CardBody
      tag="div"
    >
      <CardTitle
        tag="div"
      >
        <h3>
          Portafolio companies
        </h3>
      </CardTitle>
      <br />
      <span>
        Catalog of portafolio companies
      </span>
      <Row
        tag="div"
        widths={
          Array [
            "xs",
            "sm",
            "md",
            "lg",
            "xl",
          ]
        }
      >
        <Col
          tag="div"
          widths={
            Array [
              "xs",
              "sm",
              "md",
              "lg",
              "xl",
            ]
          }
          xs="12"
        >
          <Table
            columns={
              Array [
                Object {
                  "dataIndex": "code",
                  "key": "code",
                  "title": "Code",
                  "width": 100,
                },
                Object {
                  "dataIndex": "name",
                  "key": "name",
                  "title": "Name",
                  "width": 150,
                },
                Object {
                  "dataIndex": "number",
                  "key": "number",
                  "title": "Number Identification Fiscal",
                  "width": 100,
                },
              ]
            }
            dataSource={
              Array [
                Object {
                  "code": "CIHC",
                  "name": "CONTROLADORA IHC S.A DE C.V",
                  "number": "CIHC121455",
                },
                Object {
                  "code": "CIHC",
                  "name": "CONTROLADORA IHC S.A DE C.V",
                  "number": "CIHC121455",
                },
                Object {
                  "code": "CIHC",
                  "name": "CONTROLADORA IHC S.A DE C.V",
                  "number": "CIHC121455",
                },
                Object {
                  "code": "CIHC",
                  "name": "CONTROLADORA IHC S.A DE C.V",
                  "number": "CIHC121455",
                },
                Object {
                  "code": "CIHC",
                  "name": "CONTROLADORA IHC S.A DE C.V",
                  "number": "CIHC121455",
                },
                Object {
                  "code": "CIHC",
                  "name": "CONTROLADORA IHC S.A DE C.V",
                  "number": "CIHC121455",
                },
                Object {
                  "code": "CIHC",
                  "name": "CONTROLADORA IHC S.A DE C.V",
                  "number": "CIHC121455",
                },
                Object {
                  "code": "CIHC",
                  "name": "CONTROLADORA IHC S.A DE C.V",
                  "number": "CIHC121455",
                },
              ]
            }
            rowKey="key"
          />
        </Col>
      </Row>
      <Row
        tag="div"
        widths={
          Array [
            "xs",
            "sm",
            "md",
            "lg",
            "xl",
          ]
        }
      >
        <Col
          tag="div"
          widths={
            Array [
              "xs",
              "sm",
              "md",
              "lg",
              "xl",
            ]
          }
          xs="7"
        />
        <Col
          tag="div"
          widths={
            Array [
              "xs",
              "sm",
              "md",
              "lg",
              "xl",
            ]
          }
          xs="5"
        >
          <ButtonGroup
            role="group"
            tag="div"
          >
            <Button
              color="primary"
              tag="button"
            >
              Log
            </Button>
            <Button
              color="primary"
              tag="button"
            >
              Add
            </Button>
            <Button
              color="primary"
              tag="button"
            >
              Properties
            </Button>
            <Button
              color="primary"
              tag="button"
            >
              Add Like
            </Button>
            <Button
              color="primary"
              tag="button"
            >
              Refresh
            </Button>
            <Button
              color="primary"
              tag="button"
            >
              Close
            </Button>
          </ButtonGroup>
        </Col>
      </Row>
    </CardBody>
  </Card>
</Fragment>
`;
